lib/steps/discard_open_changes_step.go

```diff
@@ -0,0 +1,25 @@
+package steps
+
+import (
+	"github.com/Originate/git-town/lib/script"
+)
+
+type DiscardOpenChangesStep struct {
+	NoAutomaticAbort
+}
+
+func (step DiscardOpenChangesStep) CreateAbortStep() Step {
```

Comment:
These repetitive methods are another candidate for DRYing up using composition in another PR!